doc/source/language/features.rst

```diff
@@ -166,7 +165,7 @@ programming.
 But if you are an experienced programmer you might not like this
 behavior, and there are good arguments for why you might want to
 disable it.  If you wish to do so, a syntax exists to do so called
-:ref:``NOLAZYGLOBAL``.
+:ref:`@LAZYGLOBAL OFF <lazyglobal>`.
```

Comment:
Fun fact: That was the first thing that I noticed, which sent me off on this wonderful journey of learning about sphinx and RST.